python/tvm/relay/op/nn/_nn.py

```diff
@@ -129,19 +129,20 @@ def compute_conv2d(attrs, inputs, out_type, target):
     if dilation_h < 1 or dilation_w < 1:
         raise ValueError("dilation should be positive value")
 
+    def _get_channels():
```

Comment:
Is _get_out_depth() better because the returned value makes sense only for depthwise convolution?

Reply:
updated